Nguyen_Ken/Assignments/scoresAndGrades.py

```diff
@@ -0,0 +1,24 @@
+#scores and grades
+print "***Scores and Grades***"
+print "Please enter your scores"
+
+def letter(grade):
+    if (60<=grade<=69):
+        return "D"
+    elif (70<=grade<=79):
+        return "C"
+    elif (80<=grade<=89):
+        return "B"
+    elif (90<=grade<=100):
+        return "A"
+    elif (grade>100):
+        return "fabulous Glen-coco, you go Glen-coco!"
+    else:
+        return "unacceptable!"
+
+
+for i in range(1,11):
+    grade = input()
+    print "Score:" + str(grade) + "; Your grade is " + letter(grade)
+
+print "End of the program. Dueces!"
```

Comment:
Haha yeah, getting carried away. Python is fun though

Sent from Yahoo Mail on Android 
 
  On Sat, Sep 3, 2016 at 4:12 PM, authman<notifications@github.com> wrote:   
In Nguyen_Ken/Assignments/scoresAndGrades.py:
> +        return "C"
> +    elif (80<=grade<=89):
> +        return "B"
> +    elif (90<=grade<=100):
> +        return "A"
> +    elif (grade>100):
> +        return "fabulous Glen-coco, you go Glen-coco!"
> +    else:
> +        return "unacceptable!"
> +
> +
> +for i in range(1,11):
> +    grade = input()
> +    print "Score:" + str(grade) + "; Your grade is " + letter(grade)
> +
> +print "End of the program. Dueces!"

It looks like you're having fun with python 😇
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.